hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/home_work/hw01_hello_otus
```

Корректный путь до модуля будет выглядеть:
```suggestion
module github.com/prohorweb/hw-go/hw01_hello_otus
```

---

Понял, исправил.